config/local.example.php

```diff
@@ -12,4 +12,11 @@
     'ttl' => 0,
     'gearman_servers' => ['elife_gearman_1'],
     'gearman_auto_restart' => true,
+    'aws' => [
+        'mock_queue' => false,
```

Comment:
okay that would work, maybe it would be possible to set up a hybrid system where either the configuration can be set up in the file (maybe for local, I'm using a specific account for testing), or in the `.aws/credentials` file.